config/routes.rb

```diff
@@ -20,7 +20,7 @@
   end
 
   namespace :ncr do
-    resources :proposals
+    resources :work_orders
   end
```

Comment:
Can also keep the old urls if that's desirable. It was easier to have a separate system so both could be available during development, but we could switch them back now